src/jit/gentree.cpp

```diff
@@ -5476,9 +5448,7 @@ GenTree* Compiler::gtNewPhysRegNode(regNumber reg, var_types type)
 
 GenTree* Compiler::gtNewJmpTableNode()
 {
-    GenTree* node                     = new (this, GT_JMPTABLE) GenTreeJumpTable(TYP_INT);
-    node->gtJumpTable.gtJumpTableAddr = 0;
-    return node;
+    return new (this, GT_JMPTABLE) GenTree(GT_JMPTABLE, TYP_I_IMPL);
```

Comment:
hmmm... I suppose it doesn't matter that you changed the type from INT to I_IMPL

Reply:
Codegen ignores the node type so this change doesn't have any effect. Except for avoiding confusion because codegen already uses `TYP_I_IMPL`:
https://github.com/dotnet/coreclr/blob/7d54c590315ad682ed74f9491654720d83532ae7/src/jit/codegenxarch.cpp#L3734-L3739